### PR TITLE
EZP-25414 Fixed zone clone

### PR DESCRIPTION
**Pull request by yannickroger**
Link: https://jira.ez.no/browse/EZP-25414

## Description
Copy of https://github.com/ezsystems/ezflow-ls-extension/pull/2 on the right repository.

### CODE REVIEW

**Comment by yannickroger** [comment]
+1
ping @lserwatka @bdunogier @andrerom

**Comment by andrerom** [comment]
+1

File: packages/ezflow_extension/ezextension/ezflow/classes/ezpagezone.php [hunk 1]
```diff
@@ -388,6 +388,7 @@ public function __clone()
             foreach ( $this->attributes['blocks'] as $i => $block )
             {
                 $this->attributes['blocks'][$i] = clone $block;
+                $this->attributes['blocks'][$i]->setAttribute('zone_id', $this->attributes['id']);
```
Comment (bdunogier) [inline packages/ezflow_extension/ezextension/ezflow/classes/ezpagezone.php]: CS: spaces within the parenthesis.

**Comment by bdunogier** [comment]
+1

**Pull request merged**